Get custom fields (#1178)
inrupt/solid-client-access-grants-js: CHANGELOG.md

### New feature (alpha)

- Add support for custom fields. Applications are now able to read and write custom fields
  into Access Credentials (both Access Requests and Access Grants). This feature is available
  via a new option introduced in `issueAccessRequest` and `approveAccessRequest` to write the
  custom fields, and via a set of dedicated getters in the `getters/` module. A generic getter
  is introduced, `getCustomFields`, as well as a set of typed helpers, such as `getCustomInteger`
  and `getCustomIntegers`. Typed helpers are available for integers, floats, strings and booleans,

>   with a scalar and array value for each.

Reviewer: ```suggestion
  with a scalar and array version of each.
```
